src/controls/RadialProgress.tsx

```diff
@@ -38,8 +45,17 @@ class RadialProgress extends React.Component<IProps, {}> {
   }
 
   public render(): JSX.Element {
-    const { className, data, offset, style, totalRadius } = this.props;
+    const { className, data, offset, style, totalRadius, spin } = this.props;
     const sideLength = (totalRadius + (offset || 0)) * 2;
+
+    if (spin) {
+      return (
+        <svg className={`notifications-spin ${className}`} viewBox={`0 0 ${sideLength} ${sideLength}`} style={style}>
```

Comment:
It's still string concatenation and also: why is it still called **notifications**-spin?